src/containers/contact/Contact.js

```diff
@@ -1,27 +1,29 @@
 import React from "react";
 import "./Contact.css";
 import SocialMedia from "../../components/socialMedia/SocialMedia";
-import { contactInfo } from "../../portfolio";
+import {contactInfo} from "../../portfolio";
 
 export default function Contact() {
-  return (
-    <div className="main contact-margin-top" id="contact">
-      <div className="contact-div-main">
-        <div className="contact-header">
-          <h1 className="heading contact-title">{contactInfo.title}</h1>
-          <p className="subTitle contact-subtitle">{contactInfo.subtitle}</p>
-          
-          <div className="contact-text-div">
-          <h1 className="contact-detail">{contactInfo.number}</h1>
-          <h1 className="contact-detail-email">{contactInfo.email_address}</h1>
+    return (
+        <div className="main contact-margin-top" id="contact">
+            <div className="contact-div-main">
+                <div className="contact-header">
+                    <h1 className="heading contact-title">{contactInfo.title}</h1>
+                    <p className="subTitle contact-subtitle">{contactInfo.subtitle}</p>
 
-          <SocialMedia />
-          </div>
+                    <div className="contact-text-div">
+                        <a className="contact-detail" href={"tel:" + contactInfo.number}>{contactInfo.number}</a>
+                        <br/><br/>
+                        <a className="contact-detail-email"
+                           href={"mailto:" + contactInfo.email_address}>{contactInfo.email_address}</a>
+                        <br/><br/>
+                        <SocialMedia/>
+                    </div>
+                </div>
+                <div className="contact-image-div">
+                    <img alt="Saad Working" src={require("../../assests/images/contactMail.png")}></img>
+                </div>
+            </div>
```

Comment:
Try to preserve code indentation to avoid unnecessary spacing getting committed as new changes.

Reply:
I reformatted using PhpStorm. Now it **looks** clean. If you still want me to revert, I would.
![image](https://user-images.githubusercontent.com/37476886/72252055-b57fde00-3624-11ea-9307-8791714befad.png)

Reply:
You could've made a seperate commit for reformatting all code to one format of indents but it's okay now.
Just keep in mind for future contributions.
Other than that, great PR!